malariagen_data/plasmodium.py

```diff
@@ -0,0 +1,190 @@
+import json
+import os
+
+import dask.array as da
+import pandas as pd
+import xarray
+import zarr
+
+from malariagen_data.util import (
+    DIM_ALLELE,
+    DIM_PLOIDY,
+    DIM_SAMPLE,
+    DIM_VARIANT,
+    da_from_zarr,
+    init_filesystem,
+    init_zarr_store,
+)
+
+
+class PlasmodiumTools:
```

Comment:
Suggest calling this "PlasmodiumDataResource" or "PlasmodiumDataRelease", seems a bit more natural and descriptive.